cdap-common/src/main/java/co/cask/cdap/common/security/AuthEnforceRewriter.java

```diff
@@ -125,7 +128,11 @@
   private static final Type AUTHORIZATION_ENFORCER_TYPE = Type.getType(AuthorizationEnforcer.class);
   private static final Type AUTHENTICATION_CONTEXT_TYPE = Type.getType(AuthenticationContext.class);
   private static final Type ACTION_TYPE = Type.getType(Action.class);
+  private static final Type NAME_TYPE = Type.getType(Name.class);
   private static final Type AUTH_ENFORCE_UTIL_TYPE = Type.getType(AuthEnforceUtil.class);
+  private static final Set<String> SUPPORTED_PARAMETER_ANNOTATIONS =
+    Sets.newHashSet(Type.getType(Name.class).getDescriptor(), Type.getType(QueryParam.class).getDescriptor(),
+                    Type.getType(PathParam.class).getDescriptor());
 
   @Override
   public byte[] rewriteClass(String className, InputStream input) throws IOException {
```

Comment:
The `rewriteClass` is `@Nullable`. For the case that doesn't need to rewrite, better to just return `null` (line 154 of this class).